assets/memberships-gate/overlay.scss

```diff
@@ -0,0 +1,81 @@
+.newspack-memberships {
+	&__overlay-gate {
+		position: fixed;
+		top: 0;
+		left: 0;
+		right: 0;
+		bottom: 0;
+		z-index: 999999;
+		pointer-events: none;
+		overflow: hidden;
+		&::before {
+			content: '';
+			position: absolute;
+			top: 0;
+			left: 0;
+			right: 0;
+			bottom: 0;
+			background: #000;
+			opacity: 0;
+			transition: opacity 200ms ease;
+		}
+		&[data-visible='true'] {
+			pointer-events: auto;
+			&::before {
+				opacity: 0.75;
```

Comment:
What about making this configurable like we do with overlay prompts?

Reply:
I'd rather use a standard that follows the same applied to the modal checkout for a seamless transition when rendering the checkout.